docs/reference/model/file.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Could you please add documents for the file module here? https://github.com/kcl-lang/kcl-lang.io/blob/main/docs/reference/model/overview.md

Reply:
fixed